tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -468,3 +468,71 @@ async def test_keep_qsub_output(
         assert " -o /dev/null -e /dev/null" in Path("captured_qsub_args").read_text(
             encoding="utf-8"
         )
+
+
+@pytest.mark.parametrize(
+    "text_to_ignore",
+    [
+        "pbs_iff: cannot connect to host\npbs_iff: all reserved ports in use",
+        "qstat: Invalid credential",
+    ],
+)
+async def test_that_openpbs_driver_ignores_qstat_flakiness(
+    text_to_ignore: str, tmp_path, caplog, monkeypatch
+):
+    monkeypatch.chdir(tmp_path)
+    bin_path = tmp_path / "bin"
+    bin_path.mkdir()
+    monkeypatch.setenv("PATH", f"{bin_path}:{os.environ['PATH']}")
+    qsub_path = bin_path / "qsub"
+    qsub_path.write_text("#!/bin/sh\necho '1'")
+    qsub_path.chmod(qsub_path.stat().st_mode | stat.S_IEXEC)
+    qstat_path = bin_path / "qstat"
+    qstat_path.write_text(
+        "#!/bin/sh"
```

Comment:
same shell script as above??

Reply:
Yes, I was unsure where to put them, as they are split in unit_tests/integration_tests. I decided on putting them under tests/conftest as a fixture returning the `_mock_flaky_qstat`-function.